app/src/androidTest/java/org/simple/clinic/AuthenticationRule.kt

```diff
@@ -67,7 +67,7 @@ class AuthenticationRule : TestRule {
       val registrationEntry = testData.ongoingRegistrationEntry(
           phoneNumber = faker.number.number(10),
           pin = testData.qaUserPin(),
-          facilities = facilities)
+          facilities = facilities.take(1))
```

Comment:
Has the server stopped accepting multiple facilities?

Reply:
Yes, this change was merged in yesterday, it seems.

Reply:
@saket @sanchita I am going to merge this in.